tls/extensions/s2n_server_key_share.c

```diff
@@ -63,41 +73,48 @@ int s2n_extensions_server_key_share_select(struct s2n_connection *conn)
     for (uint32_t i = 0; i < ecc_pref->count; i++) {
         /* Checks supported group and keyshare have both been sent */
         if (conn->secure.client_ecc_evp_params[i].negotiated_curve &&
-             conn->secure.mutually_supported_groups[i]) {
+                conn->secure.mutually_supported_groups[i]) {
```

Comment:
Nit: is this spacing better? It seems pretty arbitrary

Reply:
This hits a tab boundary. Previously it was one space off.